lib/helpers/utils.dart

```diff
@@ -1,22 +0,0 @@
-import 'package:flutter/material.dart';
-
-/// [RouteObserver] created to remove Element in case pop in [RouteAware]
-final RouteObserver<ModalRoute<void>> routeObserver =
```

Comment:
Removing the route observers is leading to previously added elements to stay in the DOM. 

Bottom attached screen should not have any elements from previous screen but it does have

<img width="1190" alt="Screenshot 2022-02-21 at 2 40 28 PM" src="https://user-images.githubusercontent.com/26563213/154931094-23a8eb18-e24b-44ad-a689-55da487c1f83.png">

With route observers 

<img width="1190" alt="Screenshot 2022-02-21 at 2 26 19 PM" src="https://user-images.githubusercontent.com/26563213/154931517-b82af5e4-be67-4768-bed7-d75e51847a89.png">

I did not see anything related to removing elements from the DOM, how it is handled?

Reply:
Do you have any experience with SEO? I did read somewhere that hidden elements might be ignored 🤷‍♂️ 😄 

I'll take a look into using something like `visibility_detector` to hide those elements.